README.md

```diff
@@ -1,109 +1,122 @@
-
-**Notes**:
-- Whether the signature needs to be validated by the intermediaries in transit is determined by the internal implementation of each intermediary or the local schema.
-- In a future version of the API, intermediary fees may be supported; at that time, signature-at-field-level may also be supported. However, both features are out-of-scope for the current version (1.0) of the API.
+[![Git Commit](https://img.shields.io/github/last-commit/mojaloop/mojaloop-specification.svg?style=flat)](https://github.com/mojaloop/mojaloop-specification/commits/master)
+[![Git Releases](https://img.shields.io/github/release/mojaloop/mojaloop-specification.svg?style=flat)](https://github.com/mojaloop/mojaloop-specification/releases)
+[![CircleCI](https://circleci.com/gh/mojaloop/mojaloop-specification.svg?style=svg)](https://circleci.com/gh/mojaloop/mojaloop-specification)
+# Open API for FSP Interoperability Specification
+The Open API for FSP Interoperability Specification includes the following documents.
+
+#### Logical Documents
+* [Logical Data Model](#logical-data-model)
+* [Generic Transaction Patterns](#generic-transaction-patterns)
+* [Use Cases](#use-cases)
+  
+#### Asynchronous REST Binding Documents
+* [API Definition](#api-definition)
+* [Central Ledger API](#central-ledger-api) 
+* [JSON Binding Rules](#json-binding-rules) 
+* [Scheme Rules](#scheme-rules)  
+  
+#### Data Integrity, Confidentiality, and Non-Repudiation
+* [PKI Best Practices](#pki-best-practices)  
+* [Signature](#signature)  
+* [Encryption](#encryption)  
+
+## Glossary
+[This document](https://github.com/mojaloop/mojaloop-specification/blob/master/fspiop-api/documents/Glossary.md) provides the glossary for the Open API (Application Programming Interface) for FSP (Financial Service Provider) Interoperability (hereafter cited as **"the API"**). Terms have been compiled from three sources:
+- ITU-T Digital Financial Services Focus Group Glossary (ITU-T),
+- Feedback from Technology Service Providers (TSPs) in the PDP work groups (PDP) and
+- Feedback from the L1P IST Reference Implementation team (RI).
+
+Information is shared in accordance with **Creative Commons Licensing**.
+
+## Logical Data Model
+[This document](https://github.com/mojaloop/mojaloop-specification/blob/master/fspiop-api/documents/Logical%20Data%20Model.md) introduces the four generic transaction patterns that are supported in a logical version of the API. Additionally, all logical services that are part of the API are presented at a high-level.

```

Comment:
Let's use relative links here!

Reply:
And as LD said above, “this document” should be replaced by The Logical Data Model or the relevant actual document title.

Reply:
OK, will make the change. Also, it looks like the original writer of the README accidentally switched up the descriptions of the "Logical Data Model" and "Generic Transaction Patterns" documents. I will fix that too.